### PR TITLE
resolve 3.3.4

### PR DESCRIPTION
**Pull request by slubwama**

### CODE REVIEW

File: omod/src/main/webapp/resources/scripts/jquery-ui/js/openmrsSearch.js [hunk 1]
```diff
@@ -1075,14 +1075,16 @@ function OpenmrsSearch(div, showIncludeVoided, searchHandler, selectionHandler,
...
-            textToDisplay = omsgs.viewingResultsFor.replace("_SEARCH_TEXT_", "'<b>"+searchText+"</b>'");
+            if ((/^[a-zA-Z0-9_\-]+$/.test(searchText))) {
+            textToDisplay = omsgs.viewingResultsFor.replace("_SEARCH_TEXT_", "'<b>"+sanitizeHtml(searchText)+"</b>'");
             if($j.trim(searchText) == '')
                 textToDisplay = omsgs.viewingAll;
 
             $j('#pageInfo').html(sanitizeHtml(textToDisplay));
 
             if($j('#pageInfo').css("visibility") != 'visible')
                 $j('#pageInfo').css("visibility", "visible");
```
Comment (rkorytkowski) [inline omod/src/main/webapp/resources/scripts/jquery-ui/js/openmrsSearch.js:1086]: The indentation is off. Could you please adjust?

File: omod/src/main/webapp/resources/scripts/jquery-ui/js/openmrsSearch.js [hunk 2]
```diff
@@ -1075,14 +1075,16 @@ function OpenmrsSearch(div, showIncludeVoided, searchHandler, selectionHandler,
             return Math.ceil(this._table.fnSettings()._iDisplayStart / this._table.fnSettings()._iDisplayLength) + 1;
         },
         _updatePageInfo: function(searchText) {
-            textToDisplay = omsgs.viewingResultsFor.replace("_SEARCH_TEXT_", "'<b>"+searchText+"</b>'");
+            if ((/^[a-zA-Z0-9_\-]+$/.test(searchText))) {
```
Comment (rkorytkowski) [inline omod/src/main/webapp/resources/scripts/jquery-ui/js/openmrsSearch.js:1078]: Isn't it enough to sanitizeHtml? Why this "if"?

Comment (slubwama) [reply omod/src/main/webapp/resources/scripts/jquery-ui/js/openmrsSearch.js:1078]: @rkorytkowski It wasn't when I tested. I still found the same issue. Secondly Its better handled at the generation point for the HTML than at the use level of the OpenmrsSearch function.

File: omod/src/main/webapp/resources/scripts/jquery-ui/js/openmrsSearch.js (hunk 1 above)
Comment (slubwama) [reply omod/src/main/webapp/resources/scripts/jquery-ui/js/openmrsSearch.js]: @rkorytkowski  resolved

**Pull request merged**